e2e/cli_test.go

```diff
@@ -919,13 +919,13 @@ foo/main_test.rego -> wow/main_test.rego:
 
 In project root: %[1]s/v0
 main.rego:
-- opa-fmt
+- use-rego-v1
```

Comment:
This is is incorrect, and should be fixed. I'm only not sure where the issue is yet.

Reply:
It's not incorrect actually, but we'll want to find out why there is a difference :)